mrmustard/physics/fock.py

```diff
@@ -735,9 +735,7 @@ def estimate_quadrature_axis(cutoff, minimum=5, period_resolution=20):
     return xaxis
 
 
-def quadrature_distribution(
-    state: Tensor, quadrature_angle: float = 0.0, x: Vector = None, hbar: float = settings.HBAR
-):
+def quadrature_distribution(state: Tensor, quadrature_angle: float = 0.0, x: Vector = None):
```

Comment:
I would keep this as an optional argument. This allows to use the `fock` module as an independent utility module in scripts without the need of using the full config of MM. What do you think?

Reply:
We can do that too, but we can't initialize the arg using `settings.FOO` otherwise it won't change when we change the setting